addons/test_event_full/tests/test_performance.py

```diff
@@ -125,7 +125,7 @@ def test_event_create_form_notype_website(self):
         has_social = 'social_menu' in self.env['event.event']  # otherwise view may crash in enterprise
 
         # no type, website
-        with freeze_time(self.reference_now), self.assertQueryCount(event_user=735):  # tef only: 622 - com runbot: 623
+        with freeze_time(self.reference_now), self.assertQueryCount(event_user=738):  # tef only: 624 - com runbot: 624
```

Comment:
Do you have an idea on why we are triggering more queries from the event.event Form?